docs/upgrading-gatekeeper.md

```diff
@@ -16,10 +16,21 @@ and skipping `v3.4.Z`.
 
 ## 1. Set desired Gatekeeper version and commit
 
+Change ./REPLACES_VERSION and ./VERSION
+Example: 
+If you want to upgrade from 3.14.0 to 3.15.0
+***REPLACES_VERSION*** file
+```text 
+3.14.0
+```
+***VERSION*** file
+```text 
+3.15.0
+```
+
```

Comment:
Providing commands like `releasing.md` does would help here to keep users at the shell:
```suggestion
Update ./REPLACES_VERSION and ./VERSION files.

For example, if you want to upgrade from the current version to 3.15.0:
```shell
cat VERSION > REPLACES_VERSION
printf "3.15.0" > VERSION
```
```